src/ffi/mod.rs

```diff
@@ -1039,6 +1096,7 @@ mod tests {
             log_level,
             telio_logger,
         );
+        println!("what {} == {}; {}", res, TELIO_RES_OK, !telio_dev.is_null());
```

Comment:
Maybe turn this into log! or remove it ?

Reply:
Yeah, this is my leftover debug code, that I forgot to remove... So I'll clean it up.